storm-core/test/clj/org/apache/storm/cluster_test.clj

```diff
@@ -308,11 +309,11 @@
 
 (deftest test-cluster-state-default-acls
   (testing "The default ACLs are empty."
-    (stubbing [zk/mkdirs nil
+    (stubbing [Zookeeper/mkdirs nil
```

Comment:
You cannot stub a java static function.  It only works on clojure functions.  Please take a look at how we did this in #1030 and #1043.  It is not the cleanest thing, but it does work until we can move to an architecture that is much more java like.